.github/workflows/native.yml

```diff
@@ -125,9 +125,18 @@ jobs:
       - name: coverage
         run: npm run coverage
         working-directory: clients/cobol-lsp-vscode-extension
-      - name: configure server
+      - name: configure server jar
         working-directory: clients/cobol-lsp-vscode-extension
-        run: "cp -R ../../staging/. ./server/"
+        run: "cp -R ../../staging/jar/* ./server/jar"
+      - name: configure windows executable
+        working-directory: clients/cobol-lsp-vscode-extension
+        run: "cp -R ../../staging/package-win/* ./server/package-win"
+      - name: configure mac executable
+        working-directory: clients/cobol-lsp-vscode-extension
+        run: "cp -R ../../staging/package-macos/* ./server/package-macos"
+      - name: configure linux executable
+        working-directory: clients/cobol-lsp-vscode-extension
+        run: "cp -R ../../staging/package-linux/* ./server/package-linux"
```

Comment:
I suggest to be explicit with the file names that are being copied so that it is clear what we are copying.

Reply:
updated